### ISSUE TITLE
Countdown Timer

### ISSUE THREAD

**Issue by angelabyrnenz** [soflyy/oxygen-bugs-and-features#298]
Please could we have countdown timers included within Oxygen. 
https://bashooka.com/coding/jquery-css3-countdown-timer-scripts/

**Comment by INTELLIPLANS** [comment]
+1

**Comment by INTELLIPLANS** [comment]
I like this one also: https://demo.wponlinesupport.com/countdown-timer-ultimate-demo/

**Comment by geokongr** [comment]
Yes, that would be great for webinars, events, (online) meetings etc.
+1
